<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SailTaskEditorClass</class>
 <widget class="ITaskEditor" name="SailTaskEditorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>808</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SailTaskEditor</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_4">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>300</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="title">
      <string>航行任务</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="lbl_deviceID">
        <property name="text">
         <string>设备号  </string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="le_deviceID">
        <property name="text">
         <string>021001</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lbl_taskAdjustment">
        <property name="text">
         <string>任务调整</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="cb_taskCrud">
        <item>
         <property name="text">
          <string>增加</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>删除</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>修改</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lbl_taskNum">
        <property name="text">
         <string>任务序号</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDateTimeEdit" name="dte_taskNum"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lbl_taskType">
        <property name="text">
         <string>任务类型</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="cb_taskType">
        <item>
         <property name="text">
          <string>定深航行</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>定高航行</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="lbl_startPoint">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>21</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>21</height>
         </size>
        </property>
        <property name="text">
         <string>起点</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="lbl_startLon">
        <property name="text">
         <string>经度</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="le_startLon"/>
      </item>
      <item row="5" column="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>°</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="lbl_startLat">
        <property name="text">
         <string>纬度</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="le_startLat"/>
      </item>
      <item row="6" column="2">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>°</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="lbl_startDepth">
        <property name="text">
         <string>深度</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLineEdit" name="le_startdepth"/>
      </item>
      <item row="7" column="2">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>m</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="lbl_EndPoint">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>21</height>
         </size>
        </property>
        <property name="text">
         <string>终点</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="lbl_endLon">
        <property name="text">
         <string>经度</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QLineEdit" name="le_endLon"/>
      </item>
      <item row="9" column="2">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>°</string>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="lbl_endLat">
        <property name="text">
         <string>纬度</string>
        </property>
       </widget>
      </item>
      <item row="10" column="1">
       <widget class="QLineEdit" name="le_endLat"/>
      </item>
      <item row="10" column="2">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>°</string>
        </property>
       </widget>
      </item>
      <item row="11" column="0">
       <widget class="QLabel" name="lbl_endDepth">
        <property name="text">
         <string>深度</string>
        </property>
       </widget>
      </item>
      <item row="11" column="1">
       <widget class="QLineEdit" name="le_enddepth"/>
      </item>
      <item row="11" column="2">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>m</string>
        </property>
       </widget>
      </item>
      <item row="12" column="0">
       <widget class="QLabel" name="lbl_rotateSpeed">
        <property name="text">
         <string>转速        </string>
        </property>
       </widget>
      </item>
      <item row="12" column="1">
       <widget class="QLineEdit" name="le_rotateSpeed"/>
      </item>
      <item row="12" column="2">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>rpm</string>
        </property>
       </widget>
      </item>
      <item row="13" column="0">
       <widget class="QLabel" name="lbl_Speed">
        <property name="text">
         <string>速度        </string>
        </property>
       </widget>
      </item>
      <item row="13" column="1">
       <widget class="QLineEdit" name="le_Speed"/>
      </item>
      <item row="13" column="2">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>m/s</string>
        </property>
       </widget>
      </item>
      <item row="14" column="0">
       <widget class="QLabel" name="label_timeOutPeriod">
        <property name="text">
         <string>超时时间</string>
        </property>
       </widget>
      </item>
      <item row="14" column="1">
       <widget class="QLineEdit" name="le_timeOutPeriod"/>
      </item>
      <item row="14" column="2">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>s</string>
        </property>
       </widget>
      </item>
      <item row="15" column="0">
       <widget class="QLabel" name="lbl_dangerDepth">
        <property name="text">
         <string>危险深度</string>
        </property>
       </widget>
      </item>
      <item row="15" column="1">
       <widget class="QLineEdit" name="le_dangerDepth"/>
      </item>
      <item row="15" column="2">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>m</string>
        </property>
       </widget>
      </item>
      <item row="16" column="0">
       <widget class="QLabel" name="lbl_dangerHeight">
        <property name="text">
         <string>危险高度</string>
        </property>
       </widget>
      </item>
      <item row="16" column="1">
       <widget class="QLineEdit" name="le_dangerHeight"/>
      </item>
      <item row="16" column="2">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>m</string>
        </property>
       </widget>
      </item>
      <item row="17" column="0">
       <widget class="QLabel" name="lbl_obstacleDistance">
        <property name="text">
         <string>障碍物距离</string>
        </property>
       </widget>
      </item>
      <item row="17" column="1">
       <widget class="QLineEdit" name="le_obstacleDistance"/>
      </item>
      <item row="17" column="2">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>m</string>
        </property>
       </widget>
      </item>
      <item row="18" column="0">
       <widget class="QLabel" name="lbl_dangerVoltage">
        <property name="text">
         <string>危险电压</string>
        </property>
       </widget>
      </item>
      <item row="18" column="1">
       <widget class="QLineEdit" name="le_dangerVoltage"/>
      </item>
      <item row="18" column="2">
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>V</string>
        </property>
       </widget>
      </item>
      <item row="19" column="0">
       <widget class="QLabel" name="lbl_dangerCurrent">
        <property name="text">
         <string>危险电流</string>
        </property>
       </widget>
      </item>
      <item row="19" column="1">
       <widget class="QLineEdit" name="le_dangerCurrent"/>
      </item>
      <item row="19" column="2">
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string>A</string>
        </property>
       </widget>
      </item>
      <item row="20" column="0">
       <widget class="QLabel" name="lbl_isFloat">
        <property name="text">
         <string>上浮</string>
        </property>
       </widget>
      </item>
      <item row="20" column="1" alignment="Qt::AlignHCenter">
       <widget class="QCheckBox" name="cb_isFloat">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="21" column="0" colspan="2">
       <widget class="QPushButton" name="btn_saveSettings">
        <property name="text">
         <string>确认</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ITaskEditor</class>
   <extends>QWidget</extends>
   <header>itaskeditor.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
